webapp/src/main/java/com/box/l10n/mojito/entity/TMTextUnitVariant.java

```diff
@@ -73,7 +73,10 @@ public enum Status {
 
     MT_REVIEW,
     /** A string that doesn't need any work to be performed on it. */
-    APPROVED;
+    APPROVED,
+
+    /** It was translated in Mojito, so it won't be updated during third-party sync */
+    TRANSLATED_IN_MOJITO;
```

Comment:
The change looks great overall, my one concern is re naming for this new status field - and making it super clear to any users of Mojito as to what the difference is between this status and Approved (or some of the others) - as to anyone outside the team - it won't be obvious at all why some translated strings have "Approved" and why others have "Translated in mojito" as their status.

Some other options for what to name this:
MOJITO_LOCKED | MOJITO_MANAGED | OVERRIDDEN etc.?

@maallen - any thoughts on what to call this new state to further differentiate it from regular translations?

Reply:
OVERRIDDEN or MOJITO_OVERRIDE maybe? I think that makes clear that something is happening that is a special case and that some normal functionality is being superseded somehow

Reply:
Updated

Reply:
Thanks for updating, approved!